components/list/list-item.js

```diff
@@ -22,6 +22,7 @@ class ListItem extends LinkMixin(RtlMixin(LitElement)) {
 			breakpoints: { type: Array },
 			checked: { type: Boolean, reflect: true },
 			illustrationOutside: { type: Boolean, attribute: 'illustration-outside'},
+			ref: { type: String, reflect: true },
```

Comment:
Is `ref` the unique id that consumers would use?  I think it would be good to understand how this will get used, and how it's different from `id`.

Reply:
Yeah, so `ref` is what I'll return when something gets checked. I didn't want to make it `id` since that has other meaning and has to be unique per page where this can be unique per list.

The hope is that the user will set this if they are using selectable option. So they can keep track of when something is checked. [When things are able to drag around this will become super important since the ref won't change.] However, if they fail to do that I generate a unique ref which is just a number so things don't break.

When something is checked (or unchecked). It sends an event saying `ref` has been checked. The list then grabs all of these and sends an on change event the gives an array of the refs that are currently checked.